Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -1695,6 +1831,21 @@ def get_audit_management_logs_command(client, args):
     )
 
 
+def create_endpoint_context(audit_logs):
+    endpoints = []
+    for log in audit_logs:
+        endpoint_details = {
+            'ID': log.get('ENDPOINTID'),
+            'Hostname': log.get('ENDPOINTNAME'),
+            'Domain': log.get('DOMAIN')
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas)
```suggestion
            'Domain': log.get('DOMAIN'),
```